### ISSUE TITLE
Change batch_size to a limit based on bytes to match Cassandra fail threshold

### ISSUE THREAD

**Issue by rdzimmer-zz** [kairosdb/kairosdb#560]
KairosDB uses `kairosdb.queue_processor.batch_size`, which is the max number of messages it will send in a batch.  These can have varying sizes in bytes.  
Cassandra uses `batch_size_fail_threshold_in_kb`.  
I set both to numbers that _should_ be safe together, but someone can always come along and abuse the KairosDB message size to something that breaks the Cassandra threshold.  
The only way to guarantee the safety would be for KairosDB's batch size to also be based on byte size.

**Comment by brianhks** [comment]
Yes I agree, they should be based on the same thing.  The number of items was just so much easier at the time :(

Be advised if the value does go over the limit Kairos will detect it retry with a smaller batch size, it will also log a warning that it has done so.  So even if someone sends something big it will still make it in.
